features/CLI/CommandLine.java

```diff
@@ -0,0 +1,186 @@
+import java.util.Scanner;
+
+
+public class CommandLine {
+
+	public static void main(String[] args) {
+		Scanner cmdLine = new Scanner(System.in);
+		System.out.println("Hello, and welcome to Team mike's UML editor.");
+		System.out.println("To exit the program, type quit");
+		System.out.println("To see all the commands available, type help");
+
+
+		while(true)
```

Comment:
So the giant switch-if statement is here to stay, but it might be nice to factor it out into its own function so that this while loop is a lot more readable.

Reply:
Yeah, that can definitely be done. User input I think has to boil down to a giant switch statement, but this was all in main just so it could be tested. Will change in next commit.